ThePorch/src/theme/theme.js

```diff
@@ -113,29 +113,21 @@ const overrides = {
   },
   H2: {
     fontFamily: typography.sans.bebas.default,
-    fontWeight: '700',
-    fontSize: 54,
-    lineHeight: 54,
   },
   H3: {
     fontFamily: typography.sans.regular.default,
-    fontWeight: '600',
   },
   H4: {
     fontFamily: typography.sans.regular.default,
-    fontWeight: '600',
   },
   H5: {
     fontFamily: typography.sans.regular.default,
-    fontWeight: '600',
   },
   H6: {
     fontFamily: typography.sans.regular.default,
-    fontWeight: '600',
```

Comment:
Removing all of these `fontWeight`s actually changes the fonts for both platforms. For example, in your screenshots above compare to the font weights in the current app:

![Screen Shot 2020-07-01 at 2 30 42 PM](https://user-images.githubusercontent.com/103927/86283890-72e89880-bba7-11ea-91e1-e2b196550b23.png)

Do we have to remove these `fontWeight`s?

Reply:
I think I'm okay removing the font weight stuff from. the `bebas` font, just not following why we need to remove everywhere?

Reply:
We don't need to remove it everywhere. I found that when I removed the font-weight Android looked much better. I also thought we may want to see the entire location name on the campus card instead of hard setting font size and line-height.